config/config.yaml

```diff
@@ -25,6 +23,7 @@ newrelic:
   license: ____LICENSE_STRING_OF_40_CHARACTERS_____
 siren_service:
```

Comment:
@mabdh shall we use call it `app/service/server`. Siren is redundant IMO.

Reply:
The idea was to keep the config extendable in case in the future we will have another service inside the same repo. But for siren case i think that does make sense.